doc/conf.py

```diff
@@ -11,32 +11,33 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, time
+import sys
+import time
+
+import deap
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.append("..")
 #sys.path.append(os.path.abspath('_ext/'))
 
-import deap
```

Comment:
We need to import DEAP after adding ``..`` to sys.path when the package is not installed the import will fail. Maybe use ``# pylint: disable=wrong-import-position``